oe_corporate_blocks.post_update.php

```diff
@@ -12,6 +12,31 @@
  */
 function oe_corporate_blocks_post_update_20001() {
   // Delete the config.
-  \Drupal::configFactory()->getEditable('oe_corporate_blocks.data.site_switcher')
+  \Drupal::configFactory()
+    ->getEditable('oe_corporate_blocks.data.site_switcher')
     ->delete();
 }
+
+/**
+ * Only display "European Commission website" link in footer.
+ */
+function oe_corporate_blocks_post_update_20002(&$sandbox) {
```

Comment:
We can actually remove this update hook since the module does not yet have a release on branch 2.x..

Reply:
Done.